ipaserver/plugins/baseuser.py

```diff
@@ -573,8 +576,11 @@ def check_userpassword(self, entry_attrs, **options):
             setattr(context, 'randompassword', entry_attrs['userpassword'])
 
     def check_objectclass(self, ldap, dn, entry_attrs):
-        if ('ipasshpubkey' in entry_attrs or 'ipauserauthtype' in entry_attrs
-            or 'userclass' in entry_attrs or 'ipatokenradiusconfiglink' in entry_attrs):
+        if 'ipasshpubkey' in entry_attrs or \
```

Comment:
The check can be simplified with a ``set()`` operation:

```
if {'ipasshpubkey', 'ipauserauthtype', 'userclass', 'ipatokenradiusconfiglink', 'ipatokenradiususername'}.intersection(entry_attrs):
```

It's easier to read and faster. Dict's keys views are set-like objects.

Reply:
@tiran 
Thanks, I updated the PR with your suggestion.